### PR TITLE
[#5926] Replaced words to create a more inclusive environment (main)

### PR DESCRIPTION
**Pull request by korydraughn**
Currently investigating how to better express the meaning of primary and secondary in regards to the catalog.

### CODE REVIEW

File: lib/core/src/parseCommandLine.cpp [hunk 1]
```diff
@@ -64,8 +64,8 @@ parseCmdLineOpt( int argc, char **argv, const char *optString, int includeLong,
                 rodsArgs->serial = True;
                 argv[i] = "-Z";
             }
-            if ( strcmp( "--master-icat", argv[i] ) == 0 ) {
-                rodsArgs->masterIcat = True;
+            if (strcmp("--primary-icat", argv[i]) == 0) {
+                rodsArgs->primaryIcat = True;
```
[the remark this answers is not in the document]
Comment (trel) [reply lib/core/src/parseCommandLine.cpp:68]: agreed - can't think of anything that would collide at this time.

**Comment by korydraughn** [comment]
Using primary/secondary is good for now. We can provide additional documentation in the future as we discover more about these symbols.

Will run these changes through the testing environment and add the #'s once things are green.

**Review by alanking — APPROVED** [review]
Sounds good

**Comment by korydraughn** [comment]
All core tests pass except test_all_rules.py, which is okay because I'm sure it is due to the work @trel is doing in regards to deprecated rules and microservices.

Added #'s

**Comment by trel** [comment]
hrm, mine haven't landed yet...

**Comment by korydraughn** [comment]
I've rebased this PR on top of tip of main. The failed test output points to work related to the removal of rule files in the icommands repo.

**Comment by korydraughn** [comment]
After more discussion, this PR has been deemed as safe for merging. We will handle any failed tests in a separate PR.

**Comment by trel** [comment]
Agreed, please to merge.

**Comment by korydraughn** [comment]
The test failure occurs here:
https://github.com/irods/irods/blob/eb8b065310993479c266f6e4428bd3c99f77d5dd/scripts/irods/test/test_all_rules.py#L379-L380

It fails because the input file does not have the `INPUT`/`OUTPUT` lines.
```bash
$ cat ./test/rules/rulemsiStageDataObj.r
acPreProcForDataObjOpen {
  msiSortDataObj("byRescClass");
  msiStageDataObj("demoResc");
}
```

Given that `msiStageDataObj()` was removed in the previously merged PR, either this rule file needs to be adjusted or removed. We need to decide whether `msiSortDataObj()` is useful.

**Pull request merged**